gdx/src/com/badlogic/gdx/scenes/scene2d/ui/TextField.java

```diff
@@ -64,9 +63,11 @@
  * @author Nathan Sweet */
 public class TextField extends Widget implements Disableable {
 	static private final char BACKSPACE = 8;
-	static protected final char ENTER_DESKTOP = '\r';
-	static protected final char ENTER_ANDROID = '\n';
-	static private final char TAB = '\t';
+	//Windows and Mac before OS X as Enter
+	static protected final char CARRIAGE_RETURN = '\r';
+	//Unix based systems as Enter
+	static protected final char NEWLINE = '\n';
+	static protected final char TAB = '\t';
 	static private final char DELETE = 127;
 	static private final char BULLET = 149;
```

Comment:
It's a little odd that these mix `protected` and `private`. Even though I can see why some are protected, I don't know why all of these chars aren't protected; could they be used in subclasses? Particularly backspace and delete.

Reply:
@tommyettinger Yes, they could be reused in subclasses, the3 protected are used in TextArea, but not DELETE AND BULLET.
this was implemented already, i did not change them.

Reply:
@tommyettinger well, just did it now.